ee/fleetd-chrome/src/tables/Table.ts

```diff
@@ -4,29 +4,48 @@
 
 import * as SQLite from "wa-sqlite";
 
+/** Creates a single UI friendly string out of chrome tables that return multiple warnings */
+const CONCAT_CHROME_WARNINGS = (warnings: ChromeWarning[]): string => {
+  const warningStrings = warnings.map(
+    (warning) => `Column: ${warning.column} - ${warning.error_message}`
+  );
+  return warningStrings.toString();
```

Comment:
If they are returned separated by newlines, does the frontend render that properly? I think that would be easier to read.

Reply:
I'll try that out in the next day, thank you!

Reply:
Ok, so I can get it to render in the UI with `  return warningStrings.join("\n");` and a line of CSS, but the exported CSV will have "\n" in the strings, tried to use "\\n" in the CSV strings but it just prints it instead of new lines. I'll wrestle with it for a little tomorrow, but might need to decide which is better, a) commas throughout, or b) rendering new lines in UI but \n in the exported CSV.

Screenshots:
<img width="1624" alt="Screenshot 2023-09-14 at 4 47 54 PM" src="https://github.com/fleetdm/fleet/assets/71795832/d5ce25b7-cb3b-4bf6-ada0-aaff71dc6a7b">
<img width="1148" alt="Screenshot 2023-09-14 at 4 47 48 PM" src="https://github.com/fleetdm/fleet/assets/71795832/d9822c03-605e-48ca-94fa-c17c64551793">